datalad/distribution/create_sibling.py

```diff
@@ -379,9 +380,14 @@ def __call__(sshurl, name=None, target_dir=None,
         # dataset instances
         datasets = {p: Dataset(p) for p in content_by_ds}
 
+        # make sure dependencies are valid
+        for d in datasets.values():
+            _check_deps(d.repo, publish_depends)
```

Comment:
ah -- nevermind me here ... I thought I was looking at publish at that point ;)  for create-sibling obviously this logic would be appropriate ;)